public/i18n/en.json

```diff
@@ -88,5 +84,15 @@
   "ftm-administrative-region-filter": "Filter by $1",
   "ftm-administrative-region-province": "Province",
   "ftm-administrative-region-department": "Department",
-  "ftm-administrative-region-state": "State"
+  "ftm-administrative-region-state": "State",
+  "ftm-cant-find-donation-site-question": "Can't find a donation site near you?",
+  "ftm-large-donation-question": "Need logistical help or compensation to donate 500+ items of PPE?",
+  "ftm-canada-province-british-columbia": "British Columbia",
+  "ftm-canada-province-ontario": "Ontario",
+  "ftm-canada-large-donations": "Please visit the province-specific ($1) and national efforts ($2) if you are a manufacturer or supplier.",
```

Comment:
uses placeholders `$1` / `$2` to insert province / country specific links. If we add links like this for other countries we'd just need to setup the country-specific config and write copy that included similar placeholders